internal/scan-manager/scanners/loader/pom.xml

```diff
@@ -29,11 +29,8 @@
     </parent>
 
     <modelVersion>4.0.0</modelVersion>
-
     <groupId>org.wso2.security.tools.scanmanager</groupId>
     <artifactId>scan-manager-scanners-loader</artifactId>
-    <version>1.0-SNAPSHOT</version>
```

Comment:
Why is the version removed?

Reply:
Version was removed from the child POM as the version has been defined in the parent POM.